src/vorta/keyring/db.py

```diff
@@ -33,6 +33,16 @@ def get_password(self, service, repo_url):
         except peewee.DoesNotExist:
             return None
 
+    def remove_password(self, service, repo_url):
+        from vorta.store.models import RepoPassword
```

Comment:
Is there anything I overlooked that makes importing in the method this necessary? I would prefer having imports at the beginning of the file.

Reply:
I did the importing inside the method as it was the same in `set_password `and `get_password` , I don't know what's the reason !